Documentation/network/servicemesh/ingress-to-gateway/http-migration.rst

```diff
@@ -0,0 +1,189 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_gateway_http_migration:
+
+**********************
+HTTP Migration Example
+**********************
+
+In this example, we will show how to take an existing Ingress configuration and migrate it to the equivalent Gateway API resource.
+
+We will use the Cilium :ref:`gs_ingress_http` as the starting Ingress configuration. 
+The same approach will apply to other controllers, albeit each Ingress controller configuration varies.
+
+The example Ingress configuration routes traffic to backend services from the
+``bookinfo`` demo microservices app from the Istio project.
+
+Review Ingress Configuration
+============================
+
+You'll find the example Ingress definition in ``basic-ingress.yaml``.
+
+.. literalinclude:: ../../../../examples/kubernetes/servicemesh/basic-ingress.yaml
+
+This example listens for traffic on port 80, routes requests for the path ``/details`` to the ``details`` service,
+and ``/`` to the ``productpage`` service.
+
+
+Create Equivalent Gateway Configuration
+=======================================
+
+To create the equivalent Gateway configuration, you must consider the following:
+
+- Entry Point
+
+The entry point is a combination of an IP address and port through which the data plane is accessible to external clients.
+
+.. tabs::
+
+    .. group-tab:: Ingress
+        
+        Every Ingress resource has two implicit entry points -- one for HTTP and the other for HTTPS traffic. 
+        An Ingress controller provides those entry points. Typically, they will either be shared by all Ingress resources, or every Ingress resource will get dedicated entry points.
+
+        .. code-block:: shell-session
+
+            apiVersion: networking.k8s.io/v1
+            kind: Ingress
+            spec:
+              ingressClassName: cilium
+
+    .. group-tab:: Gateway API
+
+        In the Gateway API, entry points must be explicitly defined in a Gateway resource. 
+        For example, if you want the data plane to handle HTTP traffic on port 80, you need to define a listener for that traffic. 
+        Typically, a Gateway implementation provides a dedicated data plane for each Gateway resource.
+
+        .. code-block:: shell-session
+
+            apiVersion: gateway.networking.k8s.io/v1beta1
+            kind: Gateway
+            metadata:
+              name: cilium
```

Comment:
I would maybe not reccomend against naming this `cilium` this will become less clear what a gateway is when trying it out

Reply:
Changed the gateway names from `cilium` to `cilium-gateway`.